DP02_Galaxy_Cluster_Weak_Lensing.ipynb

```diff
@@ -0,0 +1,769 @@
+{
```

Comment:
<ul><li>"mean": "mean tangential and cross ellipticity". </li><li>is this an ellipticity profile or a shear profile? </li><li>Explain that the error is the standard error in the mean. I'm wonderig, the table did not have errors associated to <code>e1</code> and <code>e2</code>  that could be propagated to <code>e_t</code>  . </li><li>Perhaps also make a comment about the amplitude of the signal for this particular cluster, and explain what we can learn about clusters after calculating the shear / ellipticity profile. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<ol><li>adjusted </li><li>ellipticity profile</li><li>the shape noise is much bigger than other sources of error (e.g. measurement error, large scale structure), so using the statistical error is sufficient (and also consistency with resampling)</li><li>adjusted</li></ol><p> </p>